index.bs

```diff
@@ -3907,6 +4011,9 @@ a possibly null [=attribution source=] |sourceToAttribute|, and a possibly null
     : "<code>[=trigger debug data type/trigger-aggregate-insufficient-budget=]</code>"
     :: [=map/Set=] |body|["`limit`"] to [=allowed aggregatable budget per source=],
          [=serialize an integer|serialized=].
+    : "<code>[=trigger debug data type/trigger-aggregate-insufficient-bucket-budget=]</code>"
+    :: [=map/Set=] |body|["`limit`"] to the result of running [=get remaining aggregatable budget for matched bucket=]
```

Comment:
Shall set set this to the original allocated budget for he bucket instead? And shall we add `bucket` field as well?

Reply:
Nice catch. Done.